app/DynamicsAdapter/Fams3Adapter.Dynamics.Test/SearchRequest/SearchRequestService_Vehicle_Test.cs

```diff
@@ -0,0 +1,115 @@
+using Fams3Adapter.Dynamics.Config;
+using Fams3Adapter.Dynamics.Name;
+using Fams3Adapter.Dynamics.Person;
+using Fams3Adapter.Dynamics.SearchRequest;
+using Fams3Adapter.Dynamics.Vehicle;
+using Moq;
+using NUnit.Framework;
+using Simple.OData.Client;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Fams3Adapter.Dynamics.Test.SearchRequest
+{
+    public class SearchRequestService_Vehicle_Test
+    {
+        private Mock<IODataClient> _odataClientMock;
+        private Mock<IDuplicateDetectionService> _duplicateServiceMock;
+
+        private readonly Guid _testId = Guid.Parse("6AE89FE6-9909-EA11-B813-00505683FBF4");
+        private readonly Guid _testVehicleId = Guid.Parse("6AE89FE6-9909-EA11-1111-00505683FBF4");
+        private SearchRequestService _sut;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _odataClientMock = new Mock<IODataClient>();
+            _duplicateServiceMock = new Mock<IDuplicateDetectionService>();
+            _sut = new SearchRequestService(_odataClientMock.Object, _duplicateServiceMock.Object);
```

Comment:
What's the difference between this tests and the ones in DuplicateDetectionService.cs. Is this duplicate ?